backend/src/modules/doubles-tournament/core/domain/DoublesTournament.ts

```diff
@@ -53,6 +53,21 @@ export function createTournamentWithTeams(
 function onTournamentWithTeamsWasCreated(state: DoublesTournament | undefined, event: TournamentWithTeamsWasCreated): DoublesTournament {
   return new DoublesTournament({
     tournamentId: event.tournamentId,
-    tournamentTeams: event.tournamentTeams,
+    tournamentTeams: event.tournamentTeams.map(
+      (team) =>
+        new TournamentTeam({
+          teamId: TeamId.from(team.teamId),
+          firstTeamPlayer: team.firstTeamPlayerId,
+          secondTeamPlayer: team.secondTeamPlayerId,
+        }),
+    ),
   });
 }
+
+function fromTournamentTeam(tournamentTeam: TournamentTeam) {
+  return {
+    teamId: tournamentTeam.teamId.raw,
+    firstTeamPlayerId: tournamentTeam.firstTeamPlayer,
+    secondTeamPlayerId: tournamentTeam.secondTeamPlayer,
+  };
+}
```

Comment:
czemu tu widać jakieś zmiany z innego modułu? Czy to wtedy już z Tomkiem poprawiliście i potem Tomek dorzucił te fix'y?

Reply:
Prawdopodobnie dlatego, ze pobierałem zmiany z gałęzi Tomasza.